corehq/sql_db/routers.py

```diff
@@ -83,6 +83,8 @@ def allow_migrate(db, app_label):
         return db == partition_config.get_proxy_db()
     elif app_label == BLOB_DB_APP and db == 'default':
         return True
+    elif app_label == BLOB_DB_APP and model_name == 'blobexpiration':
```

Comment:
This does not seem like a very scalable long-term solution. I wouldn't want to use it for many different models because it would result in a lot of cruft building up in the router. Will it ever be possible to remove this?

Reply:
Initially I was hoping to use something similar to what's in the read/write method https://github.com/dimagi/commcare-hq/blob/93c8bbd90888bc227cdff8c55297ebeb1ee9799e/corehq/sql_db/routers.py#L128-L131 but that's not possible when we're removing a model, because the model no longer exists.

This is only for a short time to get through migrations on instances that were initialized before August last year https://github.com/dimagi/commcare-hq/pull/21569/files#r211515957. I plan to create a revert PR once this mergd.